example_app/app.py

```diff
@@ -60,7 +60,7 @@ def post():
 
 @app.route("/form", methods=["POST"])
 @validate(form=FormModel, query=QueryModel)
-def post():
+def post2():
```

Comment:
Worth naming as `form_post` or similar, based on the route used?
```suggestion
def form_post():
```